src/internal/operators/shareReplay.ts

```diff
@@ -5,6 +5,42 @@ import { MonoTypeOperatorFunction, SchedulerLike } from '../types';
 import { Subscriber } from '../Subscriber';
 
 /**
+ * Share source and replay specified number of emissions on subscription.
+ *
+ * This operator is a specialization of `replay` that connects to the connectable observable sequence
+ * when the number of observers goes from zero to one, and disconnects when there are no more observers.
+ *
```

Comment:
Should mention how a successfully completed source will stay cached in the shareReplayed observable forever, but an errored source can be retried

Reply:
Done